src/borg/testsuite/item.py

```diff
@@ -158,19 +157,19 @@ def test_item_optr():
     assert Item.from_optr(item.to_optr()) is item
 
 
-def test_chunk_content_equal():
-    def ccc(a, b):
-        chunks_a = [data for data in a]
-        chunks_b = [data for data in b]
-        compare1 = chunks_contents_equal(iter(chunks_a), iter(chunks_b))
-        compare2 = chunks_contents_equal(iter(chunks_b), iter(chunks_a))
-        assert compare1 == compare2
-        return compare1
-
-    assert ccc([b"1234", b"567A", b"bC"], [b"1", b"23", b"4567A", b"b", b"C"])
-    # one iterator exhausted before the other
-    assert not ccc([b"12345"], [b"1234", b"56"])
-    # content mismatch
-    assert not ccc([b"1234", b"65"], [b"1234", b"56"])
-    # first is the prefix of second
-    assert not ccc([b"1234", b"56"], [b"1234", b"565"])
+@pytest.mark.parametrize(
+    "chunk_a, chunk_b, chunks_equal",
+    [
+        ([b"1234", b"567A", b"bC"], [b"1", b"23", b"4567A", b"b", b"C"], True),  # equal
+        ([b"12345"], [b"1234", b"56"], False),  # one iterator exhausted before the other
+        ([b"1234", b"65"], [b"1234", b"56"], False),  # content mismatch
+        ([b"1234", b"56"], [b"1234", b"565"], False),  # first is the prefix of second
+    ],
+)
+def test_chunk_content_equal(chunk_a, chunk_b, chunks_equal):
+    chunks_a = [data for data in chunk_a]
+    chunks_b = [data for data in chunk_b]
```

Comment:
while we clean this up, maybe we could do a small readability improvement:

- have the test parameter lists with str members (no `b` for bytes)
- use `.encode()` to create the lists with bytes objects in 170/171